.claude/skills/pr-prepare/SKILL.md

```
@@ -0,0 +1,149 @@
+- [x] Tests pass (`uv run pytest`)
+- [x] Linting passes (`uv run ruff check`)
+
+## Notes
+
+Follow-up: Add Microsoft provider support
+```
+
+---
+
+Ready to copy! Push with:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)